templates/freifunk-files/update.sh

```diff
@@ -106,15 +113,92 @@ if [ $run_mesh = true ]; then
 	batctl if add fastd_backbone
     fi
 
-    if [ $(batctl if | grep fastd_nodes -c) = 0 ]; then
-        echo "(I) Add fastd nodes interface to batman-adv."
-        ip link set fastd_nodes up
-        ip addr flush dev fastd_nodes
+    if [ $(batctl if | grep fastd_nodes01 -c) = 0 ]; then
```

Comment:
use a loop in bash or a loop in the template

Reply:
Speziell die update.sh ist voll von so kram, den man mal besser/ganz anders machen müsste. Man könnte z.B. vieles in dem Script weglassen wenn man sich mehr auf systemd verlassen würde (services starten/am leben halten etc) . Ist halt die Frage, ob wir jetzt nur noch perfekte Dinge mergen, oder nach und nach aufräumen/Dinge besser machen.

Reply:
true, true... aber ist halt auch nen speicherplatz und qualitäts thema duplikate zu vermeiden und das ist aus meiner sicht wohl der einzige grund warum wir noch nicht gluon nehmen.

Reply:
Im neuen Commit sind jetzt in der update.sh zwei for loops, die die fastd_nodes daemons starten und dann das entsprechende setup machen.
Ausserdem gibt's in den Hostvars jetzt ne neue Variable mit der Anzahl der CPU Cores, die man verwenden möchte, die hier auch benutzt wird.
Das letzte fehlende Element wäre dann, dass man die fastd_nodes0x.conf files alle aus einem Template macht. Da schlage ich vor, dass wir das in einem weiteren Commit in einem neuen PR machen.